yourgov/components/Staff/AccessRequestsTable/AccessRequestsTable.tsx

```diff
@@ -119,16 +118,23 @@ export const AccessRequestsTable = () => {
 
 	return (
 		<>
-			{successMessageContent && successMessageType && (
+			<Box
+				css={{
+					display:
+						successMessageContent && successMessageType ? 'block' : 'none',
+				}}
+				focusRingFor="all"
+				ref={pageAlertRef}
+				tabIndex={-1}
+			>
 				<SectionAlert
-					focusOnUpdate={focusUpdateTime}
 					onClose={() => {
 						setSuccessMessageContent(undefined);
 					}}
 					title={`Access request for ${
 						Array.isArray(successMessageContent)
 							? successMessageContent.length
-							: `${successMessageContent.firstName} ${successMessageContent.lastName}`
+							: `${successMessageContent?.firstName} ${successMessageContent?.lastName}`
```

Comment:
Not for now but we'll need to find a way to safely remove the `?`. Not an issue for this implementation but it's just something we should clean up when we pattern the focus management in apps.